### DISCUSSION TITLE
Display postcodes

### DISCUSSION THREAD

**Discussion by JulianGro** [osmandapp/OsmAnd#14821] [Q&A]
Hello,
is there an option to display / show postcodes / ZIP codes?
In my area you can have the same road name multiple times in one city, so the postcode is important to double check if OSMAnd has actually found the right address.

**Comment by vshcherb** [comment]
I'm afraid there is only utility binaryInspector in OsmAndMapCreator to inspect but there is no visual way as of today

**Comment by Eiernase** [comment]
Any updates on this? I regularly need to look up addresses in different services as well just to verify post codes.
